tests/compute/discard-stmt.slang

```diff
@@ -1,5 +1,6 @@
 //TEST(compute):COMPARE_RENDER_COMPUTE: -shaderobj
 //TEST(compute):COMPARE_RENDER_COMPUTE: -mtl -shaderobj
+//TEST(compute):COMPARE_RENDER_COMPUTE: -wgsl -shaderobj
```

Comment:
I'm pretty sure this test is already running for WGSL due to line 1. Did it not run locally, for you?

Reply:
ah, oversight on my part, thanks